### DISCUSSION TITLE
Tray icons not working

### DISCUSSION THREAD

**Discussion by BioBerl** [eneshecan/whatsapp-for-linux#172] [Q&A] [answered]
Hi, the tray icon seems to not be working with Ubuntu 21.10, is this a configuration problem ?
![image](https://user-images.githubusercontent.com/47793643/153726580-ca70d52e-30d7-4812-acd8-b795a257eb13.png)

**Comment by xeco23** [comment, marked as answer]
Well, this is a known issue with snap as mentioned here #98 but we are yet far away from a proper solution...
